### PR TITLE
Add scheduled action for weekly releases, update docs with binary download

### PR DESCRIPTION
**Pull request by eatonphil**
See https://github.com/tigerbeetledb/tigerbeetle/releases/tag/0.0.0-test for example.

### CODE REVIEW

File: .github/workflows/scheduled_release.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+on:
+  schedule:
+    # Every Monday at 14:00
```
Comment (sentientwaffle) [inline .github/workflows/scheduled_release.yml:3]: UTC?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -9,13 +9,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
+
```
Comment (sentientwaffle) [inline .github/workflows/release.yml]: Is this accidental spacing? (If intentional that is fine).

File: README.md [hunk 3]
```diff
@@ -37,39 +37,33 @@ Here are a few key pages you might be interested in:
...
+Install TigerBeetle by grabbing the latest release from
+GitHub. For example, on Linux run:
 
 ```bash
-$ docker run -v $(pwd)/data:/data ghcr.io/tigerbeetledb/tigerbeetle \
-    format --cluster=0 --replica=0 /data/0_0.tigerbeetle
-info(io): creating "0_0.tigerbeetle"...
-info(io): allocating 660.140625MiB...
+$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2022-11-16-weekly/tigerbeetle-Linux-x64-2022-11-16-weekly.zip
+$ unzip tigerbeetle-Linux-x64-2022-11-16-weekly.zip
```
Comment (sentientwaffle) [inline README.md:56]: https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases says there is a way to link to "latest" but I can't find it so maybe the docs are out of date. :confused:

File: docs/deployment/single-binary.md [hunk 4]
```diff
@@ -2,9 +2,28 @@
...
+```
+
+Or on macOS run:
+
+```bash
+$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2022-11-16-weekly/tigerbeetle-macOS-x64-2022-11-16-weekly.zip
+$ unzip tigerbeetle-macOS-x64-2022-11-16-weekly.zip 
+$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle
+```
+
```
Comment (sentientwaffle) [inline docs/deployment/single-binary.md:59]: Maybe start a new section (header) here to delineate running from a release vs building from source.

File: .github/workflows/scheduled_release.yml (hunk 1 above)
Comment (eatonphil) [reply .github/workflows/scheduled_release.yml:3]: Probably, it doesn't really matter to me though.

File: README.md (hunk 3 above)
Comment (eatonphil) [reply README.md:56]: I have never been able to figure this out either without multiple API calls.

File: .github/workflows/release.yml (hunk 2 above)
Comment (eatonphil) [reply .github/workflows/release.yml]: Wasn't accidental but I guess it doesn't really matter.

File: README.md (hunk 3 above)
Comment (sentientwaffle) [reply README.md:56]: K, I guess is their docs are just out of date. :shrug:

File: docs/deployment/single-binary.md (hunk 4 above)
Comment (eatonphil) [reply docs/deployment/single-binary.md:59]: Good idea, updated.

**Review by sentientwaffle — DISMISSED** [review]

**Review by sentientwaffle — DISMISSED** [review]

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**